doc/user_and_programmer_guide/test_environment.md

```diff
@@ -0,0 +1,17 @@
+##Introduction
+
+This is a private environment. All the information about it is described in a document in Google Docs where we will update all the changes. The document has restricted access (Contact us for more information):
```

Comment:
This is a private environment for e2e testing addressed to fiware-cygnus developers.